site/docs/v1/tech/apis/_login-response-codes.adoc

```diff
@@ -27,10 +27,10 @@ include::../../../src/json/login/login-change-password-response.json[]
 ----
 
 |212
-|The user's email address has not yet been verified. The response will contain the User object that was authenticated. If the verification strategy has been set to `Form Field`, the response will contain the [field]#emailVerificationId# that was generated for the user.
+|The user's email address has not yet been verified. The response will contain the User object that was authenticated. If the `emailConfiguration.verificationStrategy` has been set to `FormField`, the response will contain the [field]#emailVerificationId# that was generated for the user.
```

Comment:
Can you change this to `[field]#emailConfiguration.verificationStrategy#`, please?